packages/custom_lint/lib/src/output.dart

```diff
@@ -0,0 +1,269 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:analyzer_plugin/protocol/protocol_common.dart';
+import 'package:analyzer_plugin/protocol/protocol_generated.dart';
+import 'package:cli_util/cli_logging.dart';
+import 'package:collection/collection.dart';
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p;
+
+/// The maximum length of any of the existing severity labels.
+const int _severityWidth = 7;
```

Comment:
It's weird to have this hardcoded I think. We should be able to easily compute this dymically

Reply:
Yea maybe, but I wanted to keep the whole output code as close to the original Dart analyzer code as possible. This way `custom_lint` has the same consistent output formatting and it is easier to keep up with changes in the Dart analyzer.